app/src/organisms/Desktop/RunProgressMeter/hooks/useRunProgressCopy.tsx

```diff
@@ -106,30 +105,22 @@ export function useRunProgressCopy({
     : (currentStepNumber! / analysisCommands.length) * 100
 
   const stepCountStr = ((): string | null => {
-    if (runStatus == null) {
+    if (isTerminalRunStatus(runStatus) || runStatus === RUN_STATUS_IDLE) {
```

Comment:
```suggestion
    if (runStatus === RUN_STATUS_IDLE) {
```
See previous comment - because we are already handling this condition, we'll never hit the terminal run case here.

Reply:
this line needs to stay as is because we no longer want any text showing when the run has finished or was canceled.

Reply:
Yep, you're right, thank you!